### PR TITLE
feat: CHIN-1547 - add new prop `showOnlyDiscountedPrice` 

### PR DESCRIPTION
**Pull request by IlitA**
https://byebuyglobaloperations.atlassian.net/browse/CHIN-1547

add new prop `showOnlyDiscountedPrice` to control widget prices

Screenshot 2022-05-30 at 20 53 06

### CODE REVIEW

File: src/components/productPrice/productPrice.jsx [hunk 1]
```diff
@@ -13,40 +13,63 @@ const ProductPrice = ({
...
+                    )}
+                    classNames={{
+                        discountPrice: classNames.discountPrice,
+                        originalPrice: classNames.originalPrice,
+                    }}
+                    discountPriceInCents={discountPriceInCents}
+                    originalPriceInCents={originalPriceInCents}
+                />
+            );
+        }
```
Comment (Nelspike) [inline src/components/productPrice/productPrice.jsx:46]: Small one here, I think we can save one indent with an early return here by flipping the condition and returning the simpler one without a discount first, as well as checking for the new boolean, something like this 👍🏼 

```
if (!discountPriceInCents || showOnlyDiscountedPrice) {
  // return the one at the bottom
}

// the rest with discounted prices
```

WDYT? 😄

**Review by Nelspike — COMMENTED** [review]
This looks great to me in general! 🎉 Just one small readability improvement so we don't have to worry about so many indents 😈

File: src/components/productPrice/productPrice.jsx (hunk 1 above)
Comment (IlitA) [reply src/components/productPrice/productPrice.jsx:46]: @Nelspike the price is different... I can have the price in a variable tho.

**Review by Nelspike — APPROVED** [review]
:shipit:

**Pull request merged**